doc/rados/operations/crush-map.rst

```diff
@@ -3,99 +3,103 @@
-The location of an OSD in terms of the CRUSH map's hierarchy is
-referred to as a ``crush location``.  This location specifier takes the
-form of a list of key and value pairs describing a position.  For
+The location of an OSD within the CRUSH map's hierarchy is
+referred to as a ``CRUSH location``.  This location specifier takes the
+form of a list of key and value pairs.  For
 example, if an OSD is in a particular row, rack, chassis and host, and
-is part of the 'default' CRUSH tree (this is the case for the vast
-majority of clusters), its crush location could be described as::
+is part of the 'default' CRUSH root (which is the case for most
+clusters), its CRUSH location could be described as::
 
   root=default row=a rack=a2 chassis=a2a host=a2a1
 
 Note:
 
 #. Note that the order of the keys does not matter.
 #. The key name (left of ``=``) must be a valid CRUSH ``type``.  By default
-   these include root, datacenter, room, row, pod, pdu, rack, chassis and host,
-   but those types can be customized to be anything appropriate by modifying
-   the CRUSH map.
+   these include ``root``, ``datacenter``, ``room``, ``row``, ``pod``, ``pdu``,
+   ``rack``, ``chassis`` and ``host``.
+   These defined types suffice for almost all clusters, but can be customized
+   by modifying the CRUSH map.
 #. Not all keys need to be specified.  For example, by default, Ceph
-   automatically sets a ``ceph-osd`` daemon's location to be
+   automatically sets an ``OSD``'s location to be
    ``root=default host=HOSTNAME`` (based on the output from ``hostname -s``).
 
-The crush location for an OSD is normally expressed via the ``crush location``
+The CRUSH location for an OSD is can be defined via the ``crush location``
 config option being set in the ``ceph.conf`` file.  Each time the OSD starts,
 it verifies it is in the correct location in the CRUSH map and, if it is not,
 it moves itself.  To disable this automatic CRUSH map management, add the
 following to your configuration file in the ``[osd]`` section::
 
   osd crush update on start = false
 
+Note that in most cases you will not need to manually configure this.
+
 
 Custom location hooks
 ---------------------
 
 A customized location hook can be used to generate a more complete
-crush location on startup.  The crush location is based on, in order
+CRUSH location on startup.  The CRUSH location is based on, in order
 of preference:
 
-#. A ``crush location`` option in ceph.conf.
+#. A ``crush location`` option in ``ceph.conf``
 #. A default of ``root=default host=HOSTNAME`` where the hostname is
-   generated with the ``hostname -s`` command.
+   derived from the ``hostname -s`` command
 
-This is not useful by itself, as the OSD itself has the exact same
-behavior.  However, a script can be written to provide additional
-location fields (for example, the rack or datacenter), and then the
+A script can be written to provide additional
+location fields (for example, ``rack`` or ``datacenter``) and the
```

Comment:
I nearly took out the whole section to avoid people thinking that they have to do this.  Really it's something that most people never need to touch.  Like manual CRUSH map editing here's a good chance of shooting oneself in the foot.

Some of these documents show traces of regional phrasing patterns, this may have been one.  I suspect that without Google you wouldn't know where I grew up ;)